Jenkinsfile

```diff
@@ -0,0 +1,69 @@
+#!groovy
+
+properties([
+    buildDiscarder(logRotator(daysToKeepStr: '20', numToKeepStr: '30')),
+
+    pipelineTriggers([pollSCM('H/15 * * * *')])
+])
+
+def test_ignition(ARCH, GOVERSION)
+{
+    node("${ARCH} && docker") {
+        def CGO = (ARCH == 'arm64') ? 1 : 0
+        withEnv(["TARGET=${ARCH}", "CGO_ENABLED=${CGO}",
+                 "GOARCH=${ARCH}", "GOVERSION=${GOVERSION}"]) {
+            stage("SCM $GOVERSION") {
+                checkout scm
+            }
+
+            stage("Build & Test $GOVERSION") {
+                sh '''#!/bin/bash -ex
```

Comment:
Also, can you move this script into the build directory? I want this to be more akin to Jenkinsfile simply calling `make integration-tests` or `./integration-tests` (plausibly usable in other environments), instead of baked into the Jenkinsfile.